modules/images-using-invoking-scripts-embedded-s2i.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/using_images/customizing-s2i-images.adoc
+
+[id="images-using-invoking-scripts-embedded-s2i_{context}"]
+== Invoking scripts embedded in an image
```

Comment:
Modules always need to start with an H1. If you need to adjust the way the heading shows up in the assembly, please modify the leveloffset instead.